store/catalog.go

```diff
@@ -91,3 +73,64 @@ func (c *Catalog) FindTable(ctx context.Context, find *catalog.TableFind) ([]*ca
 
 	return res, nil
 }
+
+// FindPK finds the primary key by PKFind. Implement the catalog.Catalog interface.
+func (c *Catalog) FindPK(ctx context.Context, find *catalog.PKFind) (*catalog.Index, error) {
+	table, err := c.store.GetTable(ctx, &api.TableFind{
```

Comment:
I am wondering if we can implement FindPK on top of FindIndex?

Reply:
Yes, but could you tell me why you think it's better?